ras_party/views/account_view.py

```diff
@@ -73,9 +73,15 @@ def put_email_verification(token):
     return make_response(jsonify(response), 200)
 
 
-@account_view.route('/resend-verification-email/<party_uuid>', methods=['GET'])
+@account_view.route('/resend-verification-email/<party_uuid>', methods=['POST'])
```

Comment:
Did you mean to change this from a `GET` to a `POST`?

Reply:
These should be posts but I needed to change frontstage to match.  All done now.